api/dp-openapi.yaml

```diff
@@ -497,6 +513,69 @@ components:
         and not directly modifyable. Not all fields are available to all users. Fields
```

Comment:
perhaps "modifiable"?

Reply:
Indeed, thank you. @srjfoo found some other description updates that I've also incorporated.